lib/galaxy/schema/history.py

```diff
@@ -0,0 +1,83 @@
+from datetime import datetime
+from typing import (
+    List,
+    Optional,
+)
+
+from pydantic import (
+    ConfigDict,
+    Field,
+    RootModel,
+)
+from typing_extensions import Literal
+
+from galaxy.schema.fields import EncodedDatabaseIdField
+from galaxy.schema.schema import (
+    CreateTimeField,
+    Model,
+    TagCollection,
+    UpdateTimeField,
+)
+
+HistorySortByEnum = Literal["create_time", "name", "update_time", "username"]
+
+
+class HistoryIndexQueryPayload(Model):
```

Comment:
Can you move this into api/histories.py ?

Reply:
Sure, but I'd say it makes more sense to keep it in the schema tbh. In schema.py we define various similar typing classes.

Reply:
schema is for reusable stuff you need to reason about outside of the API. that's never going to happen for request parameters

Reply:
WorkflowIndexQueryPayload is in this file and so is the Job one - I feel like this PR should keep it in here and we should debate this out of band and decide on all of these things at once? Seems odd to force the newest one to be different instead of falling into line.

Reply:
I talked to Sam about this, I guess it's ok, but these were all added very recently in a bit of an unfortunate move. Nothing stopping us from refactoring that later, this is not a blocker, but the missing functionality is.